src/components/App/FeaturedCard.jsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import IMG1 from '../../assets/img/demopic/1.jpg';
```

Comment:
Thêm 1 line trống ở dòng này, import ảnh thì em thử kiểu này:
```suggestion
import IMG1 from 'assets/img/demopic/1.jpg';
```